### PR TITLE
feat(locale): add Czech and Slovak locales

### PR DESCRIPTION
**Pull request by misaon**
### 📚 Description
Add Czech and Slovak locales

### 📝 Checklist

- [x] All commits follow the Conventional Commit format
- [x] The PR's title follows the Conventional Commit format

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,6 +85,7 @@ const {
 Currently available:
 - ar
 - az
+- cs
```
Comment (dargmuesli) [inline README.md:88]:
```suggestion
- cz
```

File: src/runtime/locale/cs.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { LocaleStrings } from '../types'
```
Comment (dargmuesli) [inline src/runtime/locale/cs.ts:1]: The file name should be `cz` as well, shouldn't it?

Comment (misaon) [reply src/runtime/locale/cs.ts:1]: I think "cs" is the correct form. The "cz" is for the name of the country, not the language. See e.g. language settings for CKEditor `config.language = 'cs';`

Comment (dargmuesli) [reply src/runtime/locale/cs.ts:1]: Following the ISO spec, it seems to be https://en.wikipedia.org/wiki/ISO_3166-2:CZ

Comment (misaon) [reply src/runtime/locale/cs.ts:1]: I'll leave the decision up to you, but the ISO you posted is about the country, not the language. Look at ISO 639-1, plus the generally accepted language codes are e.g. en-US or en-UK (en=language US=region). Czech Republic has cs-CZ, Slovakia has sk-SK.

Comment (dargmuesli) [reply src/runtime/locale/cs.ts:1]: Thank you for that information! Then, we can leave it as is.

**Review by dargmuesli — APPROVED** [review]
Thank you for that double contribution! <3

**Pull request merged**